packages/roosterjs-editor-types/lib/consts/DarkMode.ts

```diff
@@ -0,0 +1,12 @@
+import DefaultFormat from '../interface/DefaultFormat';
+
+export const DARK_MODE_DEFAULT_FORMAT = <DefaultFormat>{
```

Comment:
We still shouldn't be casting here. Why do you need to cast if you control the full value & full type?

Reply:
you're right, the cast no longer applies